homeassistant/components/blink4home.py

```diff
@@ -0,0 +1,273 @@
+"""
+Support for Blink4home cameras.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/blink4home/
+"""
+import asyncio
+import logging
+from datetime import timedelta
+import json
+import requests
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+
+_CONFIGURING = {}
+_LOGGER = logging.getLogger(__name__)
+
+ATTRIBUTION = 'Blink4Home camera support'
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+CONF_NETWORK_ID = 'network_id'
+
+DOMAIN = 'blink4home'
+DATA_BLINK = 'blink4home'
+
+API_URL = 'https://rest.prir.immedia-semi.com'
+CLIENT_SPECIFIER = 'Home-Assistant | '
+HEADERS = {'Content-Type': 'application/json'}
+TOKEN_HEADER = 'TOKEN_AUTH'
+UNAUTH_ACCESS = 'Unauthorized access'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.Schema({
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_NETWORK_ID, default=0): cv.positive_int,
+    })
+}, extra=vol.ALLOW_EXTRA)
+
+
+@asyncio.coroutine
+def async_setup(hass, config):
+    """Setting up the platform."""
+    blink_config = config.get(DOMAIN, {})
+
+    username = blink_config.get(CONF_USERNAME)
+    password = blink_config.get(CONF_PASSWORD)
+    network = blink_config.get(CONF_NETWORK_ID)
+    version = hass.config.as_dict()['version']
+
+    def arm_blink(call):
+        """Arm the system."""
```

Comment:
These services make it look like this should be an `alarm_control_panel` platform?